docs/content-crag/concepts/solids-pipelines/jobs.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Jobs | Dagster
+description: A job is an executable particularization of a graph that binds it to resources and configuration.
+---
+
+# Jobs
+
+A job is an executable particularization of a graph that binds it to resources and configuration.
+
+## Relevant APIs
+
+| Name                                                  | Description                                                                                                                                                                                                    |
+| ----------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <PyObject object="GraphDefinition" method="to_job" /> | The method used to create a job from a graph.                                                                                                                                                                  |
+| <PyObject object="JobDefinition" />                   | A job definition.  Includes a graph, [resources](/concepts/resources), default [configuration](/concepts/configuration/config-schema), tags, [hooks](/concepts/solids-pipelines/solid-hooks), and an executor. |
+
+Dagster is built around the observation that any data DAG typically contains a logical core of transformation (the graph), which is reusable across a set of environments or scenarios. The graph needs to be "particularized" for each environment, usually by plugging in configuration and services that are specific to that environment.
+
+A Dagster _job_ is the combination of a graph and a set of [resources](/concepts/resources) and [configuration](/concepts/configuration/config-schema).  You can create multiple jobs from the same graph.  For example, you might build a development job that executes a graph against sample data and stores outputs in local files, as well as production job that executes the same graph against production data and stores outputs in a cloud data lake.
```

Comment:
I think maybe leading with the example here might illustrate the value of jobs better